apps/meteor/app/models/server/models/LivechatRooms.js

```diff
@@ -269,9 +264,8 @@ export class LivechatRooms extends Base {
 			},
 		};
 
-		const livechatCount = Settings.findAndModify(query, null, update);
-
-		return livechatCount.value.value;
+		const livechatCount = await Settings.findOneAndUpdate(query, update, { returnDocument: 'after' });
+		return livechatCount.value;
```

Comment:
the correct return value here should be the same as before.. but since the return is not used right now, it doesn't break anything 😬 
```suggestion
		return livechatCount.value.value;
```